templates/calico-node.service

```diff
@@ -1,45 +1,46 @@
 [Unit]
 Description=calico node
-After=docker.service
-Requires=docker.service
 
 [Service]
 User=root
 Environment=ETCD_ENDPOINTS={{ connection_string }}
 PermissionsStartOnly=true
-ExecStartPre=-/usr/bin/docker rm -f calico-node
-ExecStart=/usr/bin/docker run --net=host --privileged --name=calico-node \
-  -e ETCD_ENDPOINTS={{ connection_string }} \
-  -e ETCD_CA_CERT_FILE={{ etcd_ca_path }} \
-  -e ETCD_CERT_FILE={{ etcd_cert_path }} \
-  -e ETCD_KEY_FILE={{ etcd_key_path }} \
-  -e NODENAME={{ nodename }} \
-  -e IP={{ ip }} \
-  -e NO_DEFAULT_POOLS=true \
-  -e AS= \
-  -e CALICO_LIBNETWORK_ENABLED=true \
-  -e IP6= \
-  -e CALICO_NETWORKING_BACKEND=bird \
-  -e CALICO_DISABLE_FILE_LOGGING=true \
-  -e FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
-  -e FELIX_IPV6SUPPORT=false \
-  -e FELIX_LOGSEVERITYSCREEN=info \
-  -e FELIX_HEALTHENABLED=true \
-  -e FELIX_PROMETHEUSREPORTERENABLED=true \
-  -e FELIX_PROMETHEUSREPORTERPORT=9081 \
-  -e FELIX_FLOWLOGSFILEENABLED=true \
-  -e FELIX_FLOWLOGSFILEINCLUDELABELS=true \
-  -e FELIX_FLOWLOGSFILEINCLUDEPOLICIES=true \
-  -e FELIX_FLOWLOGSENABLENETWORKSETS=true \
-  -v /var/run/calico:/var/run/calico \
-  -v /var/lib/calico:/var/lib/calico \
-  -v /lib/modules:/lib/modules \
-  -v /run/docker/plugins:/run/docker/plugins \
-  -v /var/run/docker.sock:/var/run/docker.sock \
-  -v /var/log/calico:/var/log/calico \
-  -v /opt/calicoctl:/opt/calicoctl \
+ExecStartPre=-/usr/local/sbin/charm-env --charm tigera-secure-ee conctl delete calico-node
+ExecStart=/usr/local/sbin/charm-env --charm tigera-secure-ee conctl run \
+  --net-host \
+  --privileged \
+  --env ETCD_ENDPOINTS={{ connection_string }} \
+  --env ETCD_CA_CERT_FILE={{ etcd_ca_path }} \
+  --env ETCD_CERT_FILE={{ etcd_cert_path }} \
+  --env ETCD_KEY_FILE={{ etcd_key_path }} \
+  --env NODENAME={{ nodename }} \
+  --env IP={{ ip }} \
+  --env NO_DEFAULT_POOLS=true \
+  --env AS= \
+  --env CALICO_LIBNETWORK_ENABLED=true \
+  --env IP6= \
+  --env CALICO_NETWORKING_BACKEND=bird \
+  --env CALICO_DISABLE_FILE_LOGGING=true \
+  --env FELIX_DEFAULTENDPOINTTOHOSTACTION=ACCEPT \
+  --env FELIX_IPV6SUPPORT=false \
+  --env FELIX_LOGSEVERITYSCREEN=info \
+  --env FELIX_HEALTHENABLED=true \
+  --env FELIX_PROMETHEUSREPORTERENABLED=true \
+  --env FELIX_PROMETHEUSREPORTERPORT=9081 \
+  --env FELIX_FLOWLOGSFILEENABLED=true \
+  --env FELIX_FLOWLOGSFILEINCLUDELABELS=true \
+  --env FELIX_FLOWLOGSFILEINCLUDEPOLICIES=true \
+  --env FELIX_FLOWLOGSENABLENETWORKSETS=true \
+  --mount /var/run/calico:/var/run/calico \
+  --mount /var/lib/calico:/var/lib/calico \
+  --mount /lib/modules:/lib/modules \
+  --mount /run/containerd/plugins:/run/containerd/plugins \
+  --mount /run/containerd/containerd.sock:/run/containerd/containerd.sock \
```

Comment:
These containerd paths (which were previously docker paths) look worrisome. What was calico-node doing with `/run/docker/plugins` and `/var/run/docker.sock` before? Will it still work under docker if we've mounted `/run/containerd/plugins` and `/run/containerd/containerd.sock` instead? Is calico-node smart enough to handle the difference here?